lnwallet/btcwallet/btcwallet.go

```diff
@@ -542,7 +557,20 @@ func (b *BtcWallet) ListTransactionDetails() ([]*lnwallet.TransactionDetail, err
 		txDetails = append(txDetails, detail)
 	}
 
-	return txDetails, nil
+	// Return empty transaction list, if offset is more than all transactions.
+	if offset > int32(len(txDetails)) {
+		return make([]*lnwallet.TransactionDetail, 0), nil
+	}
+
+	// Sort transactions by timestamp descending.
+	sort.Sort(sort.Reverse(byTimestamp(txDetails)))
```

Comment:
We can avoid creating the new type above by using this new function: https://golang.org/pkg/sort/#Slice